webapp/tests/mocha/unit/enketo/medic-xpath-extensions.spec.js

```diff
@@ -106,8 +112,8 @@ describe('medic-xpath-extensions', function () {
     });
 
     it('should report difference between date objects', function () {
-      const d1 = { t: 'date', v: new Date('2015-09-22') };
```

Comment:
Calling the `Date` constructor with date strings of this format creates a date with the time value set to _midnight UTC_.  This can cause weirdness depending on the current TZ when doing diffs/adds via `moment` because `moment` will do its stuff according to the date _as it is in the current TZ_.  

In our real implementation code, this is not a problem because we are either getting a string/number value (that we use `moment` to serialize into a date) or we get a `Date` from OpenRosa (where the local timezone for times [is also handled](https://github.com/eyelidlessness/openrosa-xpath-evaluator/blob/3d088a4c0a240b6abe657a574b9655c1f1ee0e86/src/openrosa-extensions.js#L567)).  So, to keep things working in these tests, I just switched to use `moment` so the local TZ is always accounted for when creating the Date values....